testGameSearch/GameViewModel.swift

```diff
@@ -12,27 +12,21 @@ import Combine
 class GameViewModel: ObservableObject {
     
     private var gameService: GameService
-    
-    @Published var game = Game()
-    
-    //getting a Missing argument for paramter 'from' in call error here
-    
+    @Published var game: Game?
     
     init() {
         self.gameService = GameService()
         
     }
     
-    var gameName: String = ""
+    @Published var gameName: String = ""
     
     func searchGames() {
-        if let game = self.game.addingPercentEncoding(withAllowedCharacters: .urlHostAllowed) {
-            getGames(by: game)
-        }
+            getGames(by: gameName)
```

Comment:
since the game name is not in the URL we don't need to use this encoding scheme; I had to remove this otherwise searching for multiple word titles like "Mario Bros" doesn't work.